src/components/confirm/confirm.component.js

```diff
@@ -2,18 +2,29 @@ import React from "react";
 import I18n from "i18n-js";
 import PropTypes from "prop-types";
 
+import OptionsHelper from "../../utils/helpers/options-helper";
```

Comment:
As the `OptionsHelper` is being removed, I think it would be safer to just declare them in this file or adopt the same approach as https://github.com/Sage/carbon/pull/4134

Reply:
Good point, thanks. Changed and added FIXME comments there.